config/default.json

```diff
@@ -96,6 +96,7 @@
         "MaxNotificationsPerChannel": 1000,
         "EnableConfirmNotificationsToChannel": true,
         "TeammateNameDisplay": "username",
+        "SearchArchivedChannels": false,
```

Comment:
Let's make this default ``true`` based on platform discussion and investigation in other products

Reply:
Done